pkg/models/protocols.go

```diff
@@ -0,0 +1,13 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 201 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// Protocol is a struct contain the device connection information
```

Comment:
It actually contains protocol specific device properties which may be used to establish a connection. So I'd say something like:

// Protocol contains device specific properties which can be used for establishing a connection with the device.